index.html

```diff
@@ -1,83 +1,88 @@
+      shortName: "trace-context",
+      format: "markdown",
+      subjectPrefix: "trace-context",
+      wg: "Distributed Tracing Working Group",
+      wgPublicList: "public-trace-context",
+      wgURI: "https://www.w3.org/2018/distributed-tracing/",
+      isPreview: true, //this will add ugly box saying it's preview
+      otherLinks: [{
+        key: 'Discussions',
+        data: [{
+          value: 'We are on Gitter.',
+          href: 'https://gitter.im/TraceContext/Lobby'
+        }]
+      }],
+      wgPatentURI: "https://www.w3.org/2004/01/pp-impl/108594/status"
+    };
+  </script>
+</head>
 
-        <section data-include="spec/40-other-protocols.md" data-include-format='markdown'></section>
+<body>
+  <section id='sotd' data-include="spec/02-sotd.md" data-include-format='markdown'></section>
+  <section id='abstract' data-include="spec/01-abstract.md" data-include-format='markdown'></section>
 
-        <section data-include="spec/50-privacy.md" data-include-format='markdown'></section>
-        <section data-include="spec/51-security.md" data-include-format='markdown'></section>
-        
-        <section class="informative">
-          <h1>Glossary</h1>
-          <dt><dfn data-lt="distributed traces">Distributed trace</dfn></dt>
-          <dd>
-            A distributed trace is a set of events, triggered as a result
-            of a single logical operation, consolidated across various
-            components of an application. A distributed traces contains
-            events that cross process, network and security boundaries.
-            A distributed trace may be initiated when someone presses a
-            button to start an action on a website - in this example, the
-            trace will represent calls made between the downstream services
-            that handled the chain of requests initiated by this button
-            being pressed.
-          </dd>
-        </section>
-  </body>
-</html>
+
+  <section data-include="spec/10-overview.md" data-include-format='markdown'></section>
+  <section data-include="spec/11-fields-semantics.md" data-include-format='markdown'></section>
+
+  <section data-include="spec/20-http_header_format.md" data-include-format='markdown'></section>
+
+  <section data-include="spec/30-processing-model.md" data-include-format='markdown'></section>
+
+  <section data-include="spec/40-other-protocols.md" data-include-format='markdown'></section>
+
+  <section data-include="spec/50-privacy.md" data-include-format='markdown'></section>
+  <section data-include="spec/51-security.md" data-include-format='markdown'></section>
+
+  <section class="informative">
+    <h1>Glossary</h1>
+    <dt><dfn data-lt="distributed traces">Distributed trace</dfn></dt>
+    <dd>
+      A distributed trace is a set of events, triggered as a result
+      of a single logical operation, consolidated across various
+      components of an application. A distributed traces contains
```

Comment:
Typo in the second sentence. "A distributed traces" should probably be "A distributed trace"

Reply:
Fixed